### PR TITLE
add optional soundSrc capability, fixes #14

### LINKED ISSUE #14: Optional Sounds when answer is selected
Add option for playing audio on `answerOption` selection

Audio is optional and scoped per `question` or per `answerOption` in **quizQuestion.js** config files. 

When scoped to a question, all selected answerOptions would play the same sound (_sound.mp3_):

```
        question: "This is my Question",
        imageSrc: "image.gif",
        soundSrc: "sound.mp3",
```
When scoped to `answerOption`, the specified sound would play when that `answerOption` is selected (and would override any `soundSrc` scoped to its `question`):

```
       answers: [
            {
                type: { njinsider: 1, finance: 1 },
                content: "$1 - $2 Million",
                imageSrc: ""
                soundSrc: "answer1Sound.mp3",
            },
```
In all cases, if the `soundSrc` attribute is not present or has a null or blank value, we skip it.  

This is using native `audioObject.play()` capes of ~modern browsers -- no additional libs or frameworks.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio

### PR DESCRIPTION
**Pull request by rogertheriault**
This is rudimentary, possibly not efficient, but a good proof of concept

Play a sound effect when a question is answered.

See the quizQuestions.js example

### CODE REVIEW

**Comment by rogertheriault** [comment]
@smcnally this works on Chrome desktop, but not (yet) in Mobile Safari. Still trying to sort out why not.

Anyway, please try it out. Currently just one sound for the whole quiz.

**Comment by rogertheriault** [comment]
Looks like the issue was the type of file. mp3 not good, m4a works great. I linked directly to something for now, I presume you'll want to replace with stuff on your Extroverting site.

File: src/api/quizQuestions.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ var quizQuestions = [
       multi: "Next »",
       format: "coolMatrix",
       imageSrc: "https://i0.wp.com/meanbusiness.com/wp-content/uploads/2017/05/Insomnia-Woman.jpg?resize=300,200",
+      soundSrc: "https://i1.theportalwiki.net/img/a/a5/Portal2_sfx_button_positive.m4a",
```
Comment (rogertheriault) [inline src/api/quizQuestions.js:19]: From https://theportalwiki.com/wiki/File:Portal2_sfx_button_positive.m4a
I just went looking for anything really :-)

Comment (smcnally) [reply src/api/quizQuestions.js:19]: Works well and As-Designed. Testing in mSafari and will merge from there --

Comment (smcnally) [reply src/api/quizQuestions.js:19]: WAD on mobile, too. Merging.

**Pull request merged**